app/js/components/identifiers.jsx

```diff
@@ -177,34 +155,36 @@ export default class Identifiers extends React.Component {
             `patient/${this.state.patient_uuid}/identifier/${this.state.preffered_identifier_uuid}`
           )
             .then((response) => {
-              (response.error) ?
-                toastr.error(response.error.message)
-                :
-                toastr.success("Identifier Created Successfully.") && this.reload();
+                if(response.error){
```

Comment:
I am not sure if code formatting is of any importance to our project, but I think you could format your code and line alignment, again, the irregular use of white spaces.

Reply:
I found this https://github.com/felixge/node-style-guide#2-spaces-for-indentation referenced in our wiki https://wiki.openmrs.org/display/docs/JavaScript+Conventions